CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ------
 
+- (`#80 <https://github.com/znicholls/silicone/pull/80>`_) Changed the names of crunchers for brevity. Changed ``lead_gas`` to ``latest_time_ratio`` and gave it documentation.
```

Comment:
```suggestion
- (`#80 <https://github.com/znicholls/silicone/pull/80>`_) Changed the names of crunchers for brevity. Also changed ``lead_gas`` to ``latest_time_ratio`` and added a notebook for this cruncher.
```

Reply:
Compromise: Changed the names of crunchers for brevity. Also changed ``lead_gas`` to ``latest_time_ratio`` and included it in ratio notebook.

Reply:
beauty